WebApplication/ClientApp/src/components/projectList.js

```diff
@@ -76,6 +77,17 @@ export class ProjectList extends Component {
     this.props.hideUploadFailed();
   }
 
+  componentDidMount() {
+    const params = window.location.search.substring(1);
+
+    if (params !== '') {
+      const parsed = JSON.parse(decodeURIComponent(params));
+      const jsPayload = JSON.parse(JSON.stringify(parsed));
```

Comment:
Wondering - why to stringify/parse? Can't we use `parsed`?

Reply:
This was leftover of previous payload, it is now fully gone